tools/rna_tools/htseq-clip/.shed.yml

```diff
@@ -0,0 +1,18 @@
+name: htseq_clip
+owner: rnateam
+description: |
+  htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets
+homepage_url: https://github.com/EMBL-Hentze-group/htseq-clip
+long_description:
+  htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets. 
+  It can be used to generate files necessary for data analysis using 
+  the companion R/Bioconductor package DEWSeq.
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip
+type: unrestricted
+categories:
+- Sequence Analysis
```

Comment:
Those needs to be toolshed categories.... You can not add arbritrairy ones.

Reply:
There are no categories for CLIP-seq or RNA-binding proteins. Then I will just keep RNA and Sequence analysis.